src/main/groovy/com/bmuschko/gradle/docker/tasks/image/Dockerfile.groovy

```diff
@@ -19,6 +19,7 @@ import org.gradle.api.DefaultTask
 import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
+import org.gradle.internal.impldep.org.apache.commons.collections.Closure
```

Comment:
This implementation uses an internal class from the Gradle API. It should be avoided if possible. Maybe that was just an incorrect import and was supposed to refer to `groovy.lang.Closure`.